src/picknik_ur_mock_hw_config/description/picknik_ur_machine_tending.xacro

```diff
@@ -23,13 +35,122 @@
     <xacro:include filename="$(find picknik_ur_base_config)/description/picknik_ur_attachments_macro.xacro"/>
     <xacro:include filename="$(arg environment_xacro)"/>
 
-    <!-- Robot: contains robot description from the world frame to the tool frame -->
-    <xacro:picknik_ur parent="world"  child="tool0" initial_positions_file="$(arg initial_positions_file)">
-      <origin xyz="0 0 0" rpy="0 0 ${pi/2}" />
-    </xacro:picknik_ur>
+  <xacro:include filename="$(find ur_description)/urdf/ur_macro.xacro" />
+  <xacro:property name="initial_positions_file" default="$(arg initial_positions_file)"/>
+
+  <link name="world" />
+  <!-- FIRST ROBOT -->
+  <!-- arm -->
+  <xacro:ur_robot
+    name="ur3e"
```

Comment:
@rwehner186 change this (and the other lined below) to a different UR type if you don't want one of each. See:
https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/blob/rolling/urdf/ur.urdf.xacro#L8